desktop/src/onionshare/strings.py

```diff
@@ -43,9 +43,10 @@ def load_strings(common, locale_dir):
     current_locale = common.settings.get("locale")
     strings = {}
     for s in translations[default_locale]:
-        if s in translations[current_locale] and translations[current_locale][s] != "":
+        try:
```

Comment:
I feel we should put the if inside try anyways. At least `if translations[current_locale][s] != ""`. Else it might show up a blank translation in case we have a blank translation 🤔

Reply:
Какдела